instructions/bundling-scripts/zcs-bundle.sh

```diff
@@ -46,6 +46,10 @@ cp -f ${repoDir}/zm-build/rpmconf/Install/install.sh
 cp -f ${repoDir}/zm-core-utils/src/libexec/zmdbintegrityreport                          ${ZCS_REL}/bin
 cp -f ${repoDir}/zm-mailbox/store/build/dist/versions-init.sql                          ${ZCS_REL}/data
 cp -f ${repoDir}/zm-build/${arch}/*.*                                                   ${ZCS_REL}/packages
+if [ "${buildType}" = "NETWORK" ]
```

Comment:
consistency of style: place "; then" at the end of this line, instead of placing "then" on the next line